target/linux/realtek/dts-5.10/rtl8382_zyxel_gs1900-16.dts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "rtl8380_zyxel_gs1900.dtsi"
+
+/ {
+	compatible = "zyxel,gs1900-16", "realtek,rtl838x-soc";
+	model = "ZyXEL GS1900-16 Switch";
```

Comment:
Just "ZyXEL GS1900-16" is enough, as far as I'm concerned.

Reply:
I removed Switch from the model name and added a better description of the UART.  I did not understand your comment regarding RAM description.